modules/stanford_profile_styles/stanford_profile_styles.module

```diff
@@ -28,3 +30,22 @@ function stanford_profile_styles_page_attachments(array &$attachments) {
   }
 
 }
+
+/**
+ * Implements hook_field_widget_form_alter().
+ */
+function stanford_profile_styles_field_widget_form_alter(&$element, FormStateInterface $form_state, $context) {
+
+  if ($context['widget']->getPluginId() == 'paragraphs') {
+    // dpm($context['widget']);
+    // dpm($element);
+    // Add the admin CSS library.
+    $element['#attached']['library'][] = 'stanford_profile_styles/admin';
+    /*
```

Comment:
@pookmish,
I'm not sure how to proceed with changing the label and adding help text.
This is what I tried. Didn't work. But I'm not sure it's the right approach because this text would probably show for all banners and not the top banner.

How do you suggest proceeding?

Reply:
To change the text you have to change the configuration. you can either do that by editing the field label in the site and running `drush csex -y` or manually change the field label in the file. https://github.com/SU-SWS/stanford_profile/blob/8.x-1.x/config/sync/field.field.node.stanford_page.su_page_banner.yml#L15

Reply:
Thanks Mike!